package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "openscope",
+  "name": "openScope",
   "version": "6.3.0-BETA",
```

Comment:
you should never directly modify this file.  the `package-lock.json` file is generated after an `npm` install.  I'd recommend deleting this file and re-running `npm install`.  That should also result in an update of the version number, which is also wrong here.

Reply:
done